app/src/main/java/com/x8bit/bitwarden/data/autofill/fido2/model/PublicKeyCredentialRequestOptions.kt

```diff
@@ -0,0 +1,15 @@
+package com.x8bit.bitwarden.data.autofill.fido2.model
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+/**
+ * Models the request options for a passkey request.
+ */
+@Serializable
+data class PublicKeyCredentialRequestOptions(
+    @SerialName("allowCredentials") val allowCredentials: List<String>,
+    @SerialName("challenge") val challenge: String,
+    @SerialName("rpId") val rpId: String,
```

Comment:
Can we name `rpId` something more specific, like `relyingPartyId`

Reply:
I thought it made more sense to use `rpId` to match the other instances in the app (such as the `rpId` in the `Fido2CredentialAutofillView` from the sdk, but I don't feel strongly about it so we can change it 👍

Reply:
That is a sdk class, which we have no control over. All the instances in the app have been specific.